.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [2.7]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
One other tweak; if you look at the way I've structured the CI builds on briefcase/toga, you'll see I run a smoke build on one platform *before* running the same build on other platforms. This is a load management strategy; in sprint situations, its helpful to run the tests on one platform and confirm they're likely to pass elsewhere, rather than start 4 parallel builds which will all fail for the same daft reason.

Reply:
It probably works :) I'll check!

Reply:
I've added Python 3.5, and I've added smoke tests that run on 3.5 first.

I chose to make the smoke tests run on both macOS and Linux. This is because if I didn't do that, I'd have run the tests on Linux Python 3.5, but not macOS Python 3.5, and then it wouldn't necessarily make sense to use matrix builds, since for Python 3.5, I'd have to carefully avoid running that version's tests on Linux to avoid duplication of effort. I can discuss more if desired.

Reply:
I get the intention of what you're saying; I still have some reservations, but it will suffice for now.

The background here - BeeWare has literally been the source of an outage on TravisCI because of the amount of load we threw at them during the PyCon US 2016 sprint :-) We haven't had an opportunity to give Github Actions a full workout, but we're obviously averse to throwing any load at a CI system that isn't almost guaranteed to pass. I'm willing to wear a little messiness in CI configurations if it avoids predictable failures (or even a significant testing backlog) during sprint conditions.

In this particular case, Rubicon is less likely to be a focus of attention during sprints, so it's probably less critical; but old habits, etc.